project/SparkBuild.scala

```diff
@@ -364,12 +364,15 @@ object Flume {
 }
 
 object DockerIntegrationTests {
+  // Ignore checksum
+  checksums in update := Nil
```

Comment:
The jar was getting a warning for checksum on maven build, and failing on sbt build... let me try removing it now that I explicitly pointed where the jar is for sbt, wich might make this change obsolete.